src/importers/zohoVaultCsvImporter.ts

```diff
@@ -14,15 +14,15 @@ export class ZohoVaultCsvImporter extends BaseImporter implements Importer {
         }
 
         results.forEach((value) => {
-            if (this.isNullOrWhitespace(value['Secret Name'])) {
+            if (this.isNullOrWhitespace(value['Password Name'])) {
```

Comment:
Let's support both for people who may have an old version of this export.

```
            if (this.isNullOrWhitespace(value['Password Name']) && this.isNullOrWhitespace(value['Secret Name'])) {
```

Reply:
I think it a good idea

Reply:
changes added